### ISSUE TITLE
Make rewrites/redirects/headers stable

### ISSUE THREAD

**Issue by timneutkens** [vercel/next.js#14184]
Currently blocked on https://github.com/vercel/vercel/pull/4355, #12801, #12608

**Comment by sebastienlabine** [comment]
this means that rewrite will support dynamic pages with getStaticProps?

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
